src/mscorlib/shared/System/Byte.cs

```diff
@@ -111,8 +111,20 @@ public static byte Parse(String s, NumberStyles style, IFormatProvider provider)
             return Parse(s, style, NumberFormatInfo.GetInstance(provider));
         }
 
+        public static byte Parse(ReadOnlySpan<char> s, NumberStyles style, IFormatProvider provider)
+        {
+            NumberFormatInfo.ValidateParseStyleInteger(style);
+            return Parse(s, style, NumberFormatInfo.GetInstance(provider));
+        }
+
         private static byte Parse(String s, NumberStyles style, NumberFormatInfo info)
         {
+            if (s == null) ThrowHelper.ThrowArgumentNullException(ExceptionArgument.s);
```

Comment:
Is there a reason why there is extra private wrapper method in some cases (like here) vs. the code is inlined in other cases, the equivalent change for short is:

```
public static short Parse(String s, NumberStyles style, IFormatProvider provider) 
{ 
    NumberFormatInfo.ValidateParseStyleInteger(style); 
    if (s == null) ThrowHelper.ThrowArgumentNullException(ExceptionArgument.s);  
    return Parse(s.AsSpan(), style, NumberFormatInfo.GetInstance(provider));
}
```